.github/workflows/app.yml

```diff
@@ -34,14 +34,18 @@ jobs:
       - uses: Swatinem/rust-cache@v1
 
       - uses: nttld/setup-ndk@v1
+        id: setup-ndk
         with:
           ndk-version: r23c
+          add-to-path: false
```

Comment:
why not adding it to the path?

Reply:
Explained in https://github.com/effektio/effektio/pull/177#discussion_r930449598